src/Extensions/ServiceFabric.Actors/ServiceFabricAgentsEventSource.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics.Tracing;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.ServiceFabric
+{
+	/// <summary>
+	/// Service Fabric event source
+	/// </summary>
+	[EventSource(Name = "Microsoft-OMEX-Logs-SfAgentsInit")] //Breaking Change: Name Changed from "Microsoft-OMEX-Logs"
+	internal sealed class ServiceFabricAgentsEventSource : EventSource
+	{
+		/// <summary>
+		/// Instance of service fabric event source
+		/// </summary>
+		public static ServiceFabricAgentsEventSource Instance { get; } = new ServiceFabricAgentsEventSource();
+
+
+		/// <summary>
+		/// Logs a service type registered ETW event.
+		/// </summary>
+		/// <param name="hostProcessId">Host process id</param>
+		/// <param name="actorType">Actor type</param>
+		[NonEvent]
+		public void LogActorTypeRegistered(int hostProcessId, string actorType)
+		{
+			if (!IsEnabled())
+			{
+				return;
+			}
+
+			LogActorTypeRegistered(hostProcessId, actorType, $"Actor host process {hostProcessId} registered actor type {actorType}");
```

Comment:
>Actor [](start = 54, length = 5)

These interpolated strings should be wrapped within `FormattableString.Invariant`. Otherwise, the behaviour will depend on the machine on which they are run. If you can't do this due to .NET constraints, you should use `string.Format(CultureInfo.InvariantCulture, ...)` instead

Reply:
Code removed